serializers/ad_serializer.rb

```diff
@@ -0,0 +1,25 @@
+class AdSerializer
```

Comment:
You can use one of the existing solutions for serialization, like [this one](https://github.com/jsonapi-serializer/jsonapi-serializer).